core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3992,6 +3971,21 @@ else if(!isDisplayNameUnique(displayName, jobName)){
             return FormValidation.ok();
         }
     }
+    private Set<String> loadItems(TaskGraphBuilder g, Handle loadJenkins) throws IOException {
+        final Set<String> loadedNames = Collections.synchronizedSet(new HashSet<String>());
+        g.requires(loadJenkins).attains(JOB_LOADED).notFatal().add("Loading items ",new Executable() {
+            public void run(Reactor session) throws Exception {
+                Collection<TopLevelItem> loadedItems =  new ArrayList<TopLevelItem>();
+                for( TopLevelItemLoader loader : getExtensionList(TopLevelItemLoader.class)){
+                    loadedItems.addAll(loader.load(Jenkins.this));
```

Comment:
@oleg-nenashev in many places where there is one failure, a job, for example won't load (can be alarming to). Not sure if that is the "right" thing but it *seems* common behavior today.